StateMachine.h

```diff
@@ -8,7 +8,11 @@
 #ifndef STATEMACHINE_H_
 #define STATEMACHINE_H_
 
+<<<<<<< HEAD
```

Comment:
This doesnt look like it should be in here, some thing github added I think?

Reply:
You are definitely right! I forgetted to delete it...